tests/nb/lock_HassGetState.yaml

```diff
@@ -0,0 +1,49 @@
+language: nb
+tests:
+  - sentences:
+      - "Er inngangsdøren låst?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: lock
+        name: "Inngangsdør"
+        state: "locked"
+    response: "Ja"
+
+  - sentences:
+      - "Er noen dører ulåst?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: lock
+        state: "unlocked"
+    response: "Ja, Bakdør"
+
+  - sentences:
+      - "Er alle dørene låst?"
+      - "er samtlige dører låst?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: lock
+        state: "locked"
+    response: "Nei, ikke Bakdør"
+
+  - sentences:
+      - "Hvilke dører er låst?"
+      - "Hva dør er låst?"
```

Comment:
`Hva dør` is not very grammatically correct... From `hva` in `_common.yaml` this is now allowed. Wrote a comment there as well. Maybe `hva` and `hvilke[n]` need to be split into?

Reply:
Some dialects allow "hva dører er låst". I thought is was ok to group them. Lets push thought the PR, and then we will consider the split.